blocks/api/test/source.js

```diff
@@ -47,10 +48,10 @@ describe( 'sources', () => {
 		it( 'should return HTML equivalent WPElement of matched element', () => {
 			// Assumption here is that we can cleanly convert back and forth
 			// between a string and WPElement representation
-			const html = '<blockquote><p>A delicious sundae dessert</p></blockquote>';
+			const html = '<blockquote><p>A delicious <b>sundae</b> dessert.</p></blockquote>';
 			const match = parse( html, sources.node() );
 
-			expect( renderToString( match ) ).toBe( `<body>${ html }</body>` );
+			expect( renderToString( valueToElement( [ match ] ) ) ).toBe( `<body>${ html }</body>` );
```

Comment:
@iseulde this test is failing. I didn't find a way to fix it. There is some issue with the `sources.node` parser. It inlines string nodes and nodes represented as an array. Once this is sorted out we should be good to do.

It also fails for this HTML fixture.

```html
<blockquote class="wp-block-pullquote alignnone">
    <p>Paragraph <strong>one</strong></p>
    <p>Paragraph two</p>
    <footer>by whomever</footer>
</blockquote>
```

Reply:
I think I fixed it :)